runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodeSession.java

```diff
@@ -159,9 +204,39 @@ public void resetRecent()
 		recentMined = 0;
 	}
 
-	public int getPerHour()
+	public int getPayDirtPerHour()
```

Comment:
You can just use lombok Getters instead of manually writing our accessors like this

Reply:
Will look into this